packages/@o3r/core/schematics/library/rules/rules.ng.ts

```diff
@@ -106,7 +86,7 @@ export function ngGenerateModule(options: NgGenerateModuleSchema & { targetPath:
           Object.entries(configFile.compilerOptions.paths).filter(([pathName, _]) => pathName !== options.name));
         configFile.compilerOptions.paths[options.packageJsonName] = [
           path.posix.join(relativeTargetPath, 'dist'),
-          path.posix.join(relativeTargetPath, 'src', 'public_api')
+          path.posix.join(relativeTargetPath, 'src', 'public-api')
```

Comment:
if we want to align with our packages, we should generate `public_api.json`.
Is this name has been also updated in the ng-packagr.json (in case of lib only)?

Reply:
In fact it is the default value generated by `@schematics/angular` generator for library. This is forwarded for the file name and for the entry in `ng-packagr.json`.
https://github.com/angular/schematics-angular-builds/blob/bb494b9b765e37b48720e364db8d3f4aedd04f23/library/schema.json#L23